### PR TITLE
docs: updating 21 migration steps

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

[D2IQ-94600](https://d2iq.atlassian.net/browse/D2IQ-94600)

## Description of changes being made

There are some other edits, but the gist is in `pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md`

Where not everything was getting deployed correctly for migrated clusters re: gitea. This makes it work.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4669.s3-website-us-west-2.amazonaws.com/

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by msdolbey** [comment]
Also, @faiq - I added a bit about the KIB builder image if you want to double check me on that. But, I feel confident about it and used it myself.

**Review by mmcelroy91 — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Review by cmastr — CHANGES_REQUESTED** [review]
Need to understand this requirement and maybe redocument as a work around before merging.

**Review by cwyl02 — APPROVED** [review]

File: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md
Comment (cwyl02) [inline pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md:219]: Maybe change this to modify `traefik-10.3.0-migration-overrides` configmap to make it a safer bet?

**Review by cwyl02 — CHANGES_REQUESTED** [review]

**Review by mmcelroy91 — APPROVED** [review]

**Review by cwyl02 — APPROVED** [review]

**Comment by msdolbey** [comment]
> Need to understand this requirement and maybe redocument as a work around before merging.

I think we're good here after a code fix for 2.1.5 and above.

**Comment by msdolbey** [comment]
Mary McElroy
[@dolbey]
 Casie is away today, so yes, I believe based on the over 100 message conversation that an agreeable resolution was achieved and we trust that you implemented it accordingly.  That’s a lot of fancy words to say, you can remove Do not merge!

**Pull request merged**
